vagrant/provisioning/roles/alfresco-change-password/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+- name: set alfresco port, if necessary
+  set_fact:
+    alfresco_port: 7070
+  when: alfresco_port is not defined
+
+- name: set alfresco protocol, if necessary
+  set_fact:
+    alfresco_protocol: "https"
+  when: alfresco_protocol is not defined
+
+- name: set alfresco URL
+  set_fact:
+    alfresco_url: "{{ alfresco_protocol }}://{{ alfresco_host }}:{{ alfresco_port }}"
+
+- name: change alfresco admin password
+  uri:
+    url: "{{ alfresco_url }}/alfresco/s/api/person/changepassword/admin"
+    method: POST
+    user: admin
+    password: "{{ old_alfresco_admin_password }}"
+    validate_certs: no
```

Comment:
@jovestefanovski , @dzdimov , this is my new approach for setting the Alfresco password.  you would call this role after alfresco, and before alfresco_site, and you would specify the old_alfresco_admin_password as an additional extra fact.